.github/workflows/build.docker.helpdesk.wolverine.yml

```diff
@@ -0,0 +1,43 @@
+name: Bulild Helpdesk.Wolverine Docker
```

Comment:
it's persistent!

Reply:
😆
fixed in https://github.com/oskardudycz/EventSourcing.NetCore/pull/220/commits/65694a22ca2539d8294f1dfe945a57d7c716e3d2